src/System.Reflection.Emit.ILGeneration/src/Configurations.props

```diff
@@ -1,11 +1,18 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <PackageConfigurations>
+      netstandard;
+      netstandard1.0;
```

Comment:
Aren't you harvesting netstandard1.0?

Reply:
Not for source, we need a not-supported assembly.